.github/workflows/build-verification.yml

```diff
@@ -38,6 +38,20 @@ jobs:
       - name: Run sanityCheck
         run: ./gradlew sanityCheck
 
+  unit_tests:
```

Comment:
these are not all unit tests for example `PluginApplicationTest`

Reply:
so simply `tests` or do you have a naming suggestion?

Reply:
i think tests is fine